upstream-community-operators/hpa-operator/hpa-operator.v0.1.6.clusterserviceversion.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: hpaoperator.banzaicloud.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: hpa-operator.v0.1.6
+  namespace: placeholder
+  annotations:
+    capabilities: Full Lifecycle
+    categories: Security
+    certified: "false"
+    description: Autoscale deployments / replica-sets by annotations
+    containerImage: banzaicloud/hpa-operator:0.1.6
+    repository: https://github.com/banzaicloud/hpa-operator
+    createdAt: "2019-04-09T08:00:00Z"
+    support: Banzai Cloud
+spec:
+  displayName: HPA Operator
+  description: |
+    The HPA Operator simplifies deployment autoscaling, watching for your Deployment or StatefulSet and automatically creating an HorizontalPodAutoscaler resource, should you provide the correct autoscale annotations.
+
+    All annotations must contain the `autoscaling.banzaicloud.io` prefix. It is required to specify minReplicas/maxReplicas and at least one metric to be used for autoscale.
+    You can add *Resource* type metrics for cpu & memory and *Custom* type metrics.
+    Autoscale annotations can be placed on Deployment / StatefulSet:
+
+         ```
+          apiVersion: extensions/v1beta1
+          kind: Deployment
+          metadata:
+            name: example
+            labels:
+            annotations:
+              hpa.autoscaling.banzaicloud.io/minReplicas: "1"
+              hpa.autoscaling.banzaicloud.io/maxReplicas: "3"
+              cpu.hpa.autoscaling.banzaicloud.io/targetAverageUtilization: "70"
+          ```
+
+    Examples of supported metric types:
+
+      - pod resource metrics: cpu & memory
+
+        ``cpu.hpa.autoscaling.banzaicloud.io/targetAverageUtilization: "{targetAverageUtilizationPercentage}"``
+
+        ``memory.hpa.autoscaling.banzaicloud.io/targetAverageUtilization: "{targetAverageUtilizationPercentage}"``
+
+      - custom metrics based on `Prometheus` queries:
+
+        ``
+        prometheus.customMetricName.hpa.autoscaling.banzaicloud.io/query: "sum({kubernetes_pod_name=~"^YOUR_POD_NAME.*",__name__=~"YOUR_PROMETHUES_METRICNAME"})"
+        prometheus.customMetricName.hpa.autoscaling.banzaicloud.io/targetValue: "{targetValue}"
+        prometheus.customMetricName.hpa.autoscaling.banzaicloud.io/targetAverageValue: "{targetAverageValue}"
+        ``
+
+  keywords: ['hpa', 'autoscale', 'banzaicloud', 'open source']
+  version: 0.1.6
+  maturity: alpha
+  maintainers:
+  - name: Banzai Cloud
+    email: info@banzaicloud.com
+  provider:
+    name: Banzai Cloud
+  labels:
+    alm-owner-hpaoperator: hpaoperator
+    operated-by: hpaoperator
+  selector:
+    matchLabels:
+      alm-owner-hpaoperator: hpaoperator
+      operated-by: hpaoperator
+  links:
+  - name: Blog
+    url: https://banzaicloud.com/blog/k8s-hpa-operator/
+  - name: Documentation
+    url: https://github.com/banzaicloud/hpa-operator/blob/master/README.md
+  - name: Hpa Operator Source Code
+    url: https://github.com/banzaicloud/hpa-operator
+  icon:
+  - base64data:
```

Comment:
The base64 for the icon must be on the same line as the key

Reply:
fixed